rbac/api/cross_access/util.py

```diff
@@ -44,12 +45,16 @@ def check_cross_request_expiry():
     logger.info("Completed clean up of %d cross-account requests, %d expired.", len(cars), len(expired_cars))
 
 
-def create_cross_principal(target_account, user_id):
+def create_cross_principal(target_account, user_id, target_org=None):
```

Comment:
Should we update here target_account=None? So when the flag flipped the account is not required

Reply:
Good catch. I updated the header to make target_account default to None.